articles/libraries/lock/v11/migration-guide.md

```diff
@@ -0,0 +1,51 @@
+---
+section: libraries
+title: Migrating to Lock v11
+description: How to migrate to Lock v11
+---
+# Migrating to Lock v11
+
+[Lock 11](/libraries/lock) is designed for [embedded login** scenarios](/guides/login/centralized-vs-embedded). It operates with enhanced security and removes dependencies that have been deprecated as per Auth0's roadmap. In some cases, these security enhancements may impact application behavior when upgrading from an earlier version of Lock. 
+
+## Should I migrate to v11?
+
+Lock can be used to implement authentication in different ways:
+
+- In your application, to implement embedded login.
+
+- In the [Hosted Login Page](/hosted-pages/login), where you can use the configure Lock to behave in the way you need. 
+
+Depending on how you are using Lock, you have different options:
+
+| **Scenario** | **Migration to v9** | 
+| --- | --- | 
+| In your application, to implement embedded login | Required |
```

Comment:
Similarly to the Auth0.js migration docs, I'd emphasize just a tad more the alternative of moving to centralized login instead of upgrading Lock, by replacing

> Required

with

> Required (read below)

coupled with the other edit to the warning panel